docs/user_guide/validation.md

```diff
@@ -39,9 +39,10 @@ With *right click* on the error a menu is opened with the following options:
 - Copy (to copy the message text)
 
 Automatic pan, zoom and highlight features or coordinates are performed by clicking on the result tables entry.
+On automatic pan and zoom, the coordinates are taken into account if they are provided by ili2db, if not, then the geometry of the feature (according to the OID provided by ili2db). On automatic zoom on the features geometry, it's extent is taken and on coordinates an extend of 10 map units instead.
 
 !!! Note
-    On automatic pan it pans to the feature geometry centroid, if the OID is provided by ili2db, and if not then no pan is performed (no pan to coordinates yet). On automatic zoom it zooms to the features geometry, if the OID is provided by ili2db, and if not then to the coordinates with the extend of 10 map units.
+    Since ili2db sometimes on non-geometry errors provides the coordinates as well, there could be a confusion when it zooms or pans to the coordinates there. Still it's preverable to not zoom or pan to the coordinates on geometry errors when they provide an OID. Currently the validator cannot make a difference between those error-types.
```

Comment:
```suggestion
    Since ili2db sometimes on non-geometry errors provides the coordinates as well, there could be a confusion when it zooms or pans to the coordinates there. Still it's preferable to not zoom or pan to the coordinates on geometry errors when they provide an OID. Currently the validator cannot make a difference between those error-types.
```